scripts/specs/uploadToCDN.spec.js

```diff
@@ -35,10 +35,7 @@ describe("uploadToCDN", () => {
     expect(logger.info).toHaveBeenCalledWith("Building files for CDN");
     expect(exec).toHaveBeenCalledWith("build", "pnpm run build");
     expect(logger.info).toHaveBeenCalledWith("Uploading files to CDN.");
-    expect(exec).toHaveBeenCalledWith(
-      "sftp",
-      'echo "-mkdir 1.2.3\ncd 1.2.3\nput ./dist/alloy.js\nput ./dist/alloy.min.js\nbye\n" | sftp -oHostKeyAlgorithms=+ssh-dss -oStrictHostKeyChecking=no -b - sshacs@dxresources.ssh.upload.akamai.com:/prod/alloy',
-    );
+    expect(exec).toHaveBeenCalledWith("sftp", expect.any(String));
```

Comment:
Are there any specific commands/arguments we want to ensure will be in the string or is any string cool?